src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -99,6 +99,26 @@ public PeriodicBackupRunner(DocumentDatabase database, ServerStore serverStore)
             return taskStatus == TaskStatus.Disabled ? null : GetNextBackupDetails(configuration, backupStatus, skipErrorLog: true);
         }
 
+        private DateTime? GetNextWakeupTime(long lastEtag, PeriodicBackupConfiguration configuration, PeriodicBackupStatus backupStatus)
+        {
+            // we will always wake up the database for a full backup.
+            // but for incremental we will wake the database only if there were changes made.
+            
+            var now = SystemTime.UtcNow;
```

Comment:
This is wrong, back up times are always local, not UTC

Reply:
We have to keep this time as utc (cluster wide) and only later, on the node itself we translate it to local.

Reply:
I don't follow, this is used only on the local node.

Reply:
The `backupStatus` is in UTC